src/cpu-kernels/reducers.cpp

```diff
@@ -1359,8 +1359,8 @@ ERROR awkward_reduce_argmin_bool_64(
   const int64_t* parents,
   int64_t lenparents,
   int64_t outlength) {
-  for (int64_t k = 0;  k < outlength;  k++) {
-    toptr[k] = -1;
+  for (int64_t i = 0; i < outlength;  i++) {
+    toptr[i] = -1;
   }
   for (int64_t i = 0;  i < lenparents;  i++) {
     int64_t parent = parents[i];
```

Comment:
Is there a reason for changing the dummy variable in this kernel? The fact that `k` is distinct from `i` is useful for keeping track of the fact that I'm iterating over different things, though they are in different scopes and a compiler doesn't care.

Reply:
The Specification -> CUDA compiler assumes that i is the outermost for-loop, j is the one inside that and in the future, k is the one inside both. Should I generalise it? - It is probably not so difficult to do so.

Reply:
It would be much better to keep that general. The names of the dummy variables are given to you by the example code; there's no reason to change them.

Reply:
This has been fixed now.